website/docs/concepts2/mutations.mdx

```diff
@@ -37,12 +37,34 @@ final addTodo = Mutation<Todo>();
 Typically, this variable will either be global or as a `static final` variable on
 a [Notifier].
 :::
+You can additionally choose to call the mutation like a function,
+and provide it with a key not unlike a family provider.
+
+This is useful if you want to have multiple instances of the same mutation,
+such as deleting a specific item in a list:
+```dart
```

Comment:
Could you extract those snippets to a dedicated .dart file? Cf other docs. Looks like I forgot to do it in this one.

Reply:
How's that?